horqrux/utils.py

```diff
@@ -71,3 +103,22 @@ def equivalent_state(state: Array, reference_state: str) -> bool:
 
 def overlap(state: Array, projection: Array) -> Array:
     return jnp.real(jnp.dot(jnp.conj(state.flatten()), projection.flatten()))
+
+
+def uniform_state(
+    n_qubits: int,
+) -> Array:
+    state = jnp.ones(2**n_qubits, dtype=jnp.complex128)
+    state = state / jnp.sqrt(jnp.array(2**n_qubits))
```

Comment:
Is jax by default 32 or 64 bit precision? If 32 then this operation here is most likely also cutting the precision of the state to 32.

Reply:
thanks, changed